RELEASING.md

```diff
@@ -61,7 +61,7 @@ name from above.  For example:
 ```
 
 When that PR is merged, the promoter bot will copy the images from staging to
-the final prod location (e.g. `k8s.gcr.io/git-sync/git-sync:v3.3.2`).
+the final prod location (e.g. `registry.k8s.io/git-sync/git-sync:v3.3.2`).
```

Comment:
There is one further reference above in relation to the pull request release process. It looks like that path is still valid in the `k8s.io` repository so we might need to make a further change at a later date to change that reference once the dependency has been updated?